draft-ietf-quic-transport.md

```diff
@@ -1728,11 +1727,11 @@ reordering can mean that the largest received offset on a stream can be greater
 than the total size of data received on that stream.  Receiving STREAM frames
 might not increase the largest received offset.
 
-The data sent on a stream MUST NOT exceed the largest maximum stream data value
-advertised by the receiver.  An endpoint MUST terminate a connection with a
-FLOW_CONTROL_ERROR error if it receives more data than the largest maximum
-stream data that it has sent for the affected stream, unless this is a result of
-a change in the initial limits (see {{zerortt-parameters}}).
+The largest offset sent on a stream MUST be less than the largest maximum stream offset
```

Comment:
So I get MAX_STREAM_DATA with Maximum Stream Offset of 10.  I then send 10 octets (0..9) in one STREAM frame.  The maximum offset is therefore 9.  The empty STREAM frame I then send has an offset of 10, which would appear to violate this rule.